content/en/docs/how-tos/cluster-claim.md

```diff
@@ -3,62 +3,223 @@ title: "Creating a Cluster Pool"
+  baseDomain: dptp-demo.openshift.org # the base domain to install the cluster
   imageSetRef:
     name: ocp-4.7.0-amd64 # the name of the imageSet which determines the image to install the cluster
   installConfigSecretTemplateRef:
-    name: install-config-aws-us-east-1 # the name of the secret with an installation config for the installer
+    name: dptp-demo-install-config # the name of the secret with an installation config for the installer
   skipMachinePools: true
   platform:
     aws:
       credentialsSecretRef:
-        name: hive-aws-credentials # the name of the secret with the credentials of the cloud account
+        name: demo-aws-credentials # the name of the secret with the credentials of the cloud account
       region: us-east-1
   pullSecretRef:
-    name: pull-secret # the name of the pull secret used by installer
+    name: pull-secret
   size: 1 # the number of clusters that Hive should keep provisioned and waiting for use.
-  maxSize: 10 # the maximum number of clusters that have not been destroyed.
+  maxSize: 10 # the maximum number of clusters that can exist at the same time.
 ```
 
-Hive maintains the number of clusters in the pool as specified by its `size`. A provisioned cluster will be [hibernating](https://github.com/openshift/hive/blob/master/docs/hibernating-clusters.md) after staying idled for sometime and can be waken up if a job claims it. Hive removes it from the pool once it is claimed and creates a new cluster to maintain the pool's `size`. The cluster will be destroyed after 4 hours since it is claimed. All clusters before destruction consume the quotas of the cloud account. If several jobs claim a cluster simultaneously, Hive will feed all of them up until the number of living clusters reaches the pool's `maxSize`. The meaning of other fields in the pool specification is available at [Hive's documentation](https://pkg.go.dev/github.com/openshift/hive/apis@master/hive/v1#ClusterPool).
+Pay attention to the following stanzas:
 
-It is suggested to put the relevant manifests to the cluster pool in the same `namespace`. Hive [replaces](https://github.com/openshift/hive/blob/master/docs/clusterpools.md#install-config-template) `baseDomain` with the value taken from the pool's spec and `metadata.name` with a random string in the installation config from `secret/install-config-aws-us-east-1` and then passes it onto the OpenShift installer. Customizations such as the number of workers and the type of instances can be made there. A `ClusterImageSet` which a cluster level CRD defines the production and its version of the installed cluster.
+1. `metadata.labels`: These labels will be used by jobs to specify what cluster they want to obtain for their testing.
+2. `spec.baseDomain`: A base domain for all clusters created in this pool. You usually create this domain yourself when
+   configuring the cloud platform account (see [prerequisites](#prerequisites)).
```

Comment:
You usually create this domain yourself when
   configuring the cloud platform account

--->

This domain is managed by the account on the cloud platform.

Reply:
I'd rather not. 1) Active case is generally better readable 2) I really want to stress out that the user is responsible for this ("is managed" can mean anything).

Reply:
In OpenShift doc, it is

https://docs.openshift.com/container-platform/4.7/installing/installing_aws/installing-aws-account.html

For AWS:

> Create a public hosted zone for your domain or subdomain.

https://docs.openshift.com/container-platform/4.7/installing/installing_azure/installing-azure-account.html#installation-azure-network-config_installing-azure-account

For Azure:

> create a public hosted zone for your domain or subdomain

https://docs.openshift.com/container-platform/4.7/installing/installing_gcp/installing-gcp-account.html

For GCP:

> Create a public hosted zone for your domain or subdomain in your GCP project.

For the above clouds, `create a public hosted zone for your domain` sounds like more accurate description.
But then we might need to explain the zone? That is why i got "is managed".
Maybe in your definition: create a public hosted zone for your domain = create your domain?

Reply:
I changed to sentence to what you suggest ("created a public hosted zone for your domain")